index.html

```diff
@@ -0,0 +1,194 @@
+                <li><img src="img/Counter.png" alt="" class="dot"></li>
+                <li><a href="#" class="project-link">2015</a></li>
+            </ul>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <ul class="project-languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                
+             </ul>
+             <button class="btn"><div class="senta">See Project<div></button>
+            
+        </div>
+
+        <div class="project-1">
+            <div class="project-img">
+                <a href="#"><img src="img/work-4.png" alt="project-showcase"></a>
+            </div>
+            <div class="project-name"><h2>Multi-Post Stories</h2></div>
+            <ul class="project-links">
+                <li><a href="#" class="project-link">Canopy</a></li>
+                <li><img src="img/Counter.png" alt="" class="dot"></li>
+                 <li><a href="#" class="project-link">Back End Dev</a></li>
+                <li><img src="img/Counter.png" alt="" class="dot"></li>
+                <li><a href="#" class="project-link">2015</a></li>
+
+            </ul>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <ul class="project-languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                
+             </ul>
+             <button class="btn"><div class="senta">See Project<div></button>
+
+        </div>
+
+        <div class="project-1">
+            <div class="project-img">
+                <a href="#"><img src="img/work-1.png" alt="project-showcase"></a>
+            </div>
+            <div class="project-name"><h2>Tonic</h2></div>
+            <ul class="project-links">
+                <li><a href="#" class="project-link">Canopy</a></li>
+                <li><img src="img/Counter.png" alt="" class="dot"></li>
+                 <li><a href="#" class="project-link">Back End Dev</a></li>
+                <li><img src="img/Counter.png" alt="" class="dot"></li>
+                <li><a href="#" class="project-link">2015</a></li>
+
+            </ul>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <ul class="project-languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                
+             </ul>
+             <button class="btn"><div class="senta">See Project<div></button>
+             
+        </div>
+        </section>
+
+        <section class="about-me">
+        <div class="about-me-container">
+            <section class="about-1">
+                 <h1 class="about-header">About Myself</h1>
```

Comment:
Figma 
![image](https://user-images.githubusercontent.com/78251927/229295794-9f225ce3-3570-4eed-9a8b-997e4de9d4f8.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/229295882-3626315d-f6f7-44eb-8abf-059c49318e05.png)

[OPTIONAL] 
- It would be nice if we remove the spacing at the edges of the `About` section.
- According to the template on Figma, the section is supposed to cover the entire width. 👍
- Let's increase the Font size of the text `About Myself` to match the template on Figma